Made mobile navbar look better

RameezIbrahimOfficial opened the pull request:
Re-Styled the mobile navbar

RameezIbrahimOfficial committed:
Made mobile navbar look better

KriteshTimsina reviewed (changes requested):
Overall, the UI looks great but here are some issues to be fixed
- remove background from navbar in large screen when dark mode is on
![image](https://github.com/KriteshTimsina/sastobazaar-ecommerce/assets/103115789/a5ef07af-2d86-4a23-8265-445b751c7b96)
- In mobile, clicking on links should go to that page and also close the sidebar
https://github.com/KriteshTimsina/sastobazaar-ecommerce/assets/103115789/5d30d105-4f65-4625-9fde-575585a8de0f
-Some transition would be a great add-on

RameezIbrahimOfficial committed:
✅ Removed background from navbar in large screen when dark mode is on
✅ In mobile, clicking on links should go to that page and also close the sidebar
